shared_linux/TestStandCommHost/TestStandCommHost.cxx

```diff
@@ -0,0 +1,60 @@
+#include <stdio.h>
+#include "TestStandCommHost.h"
+
+#define GET_BYTE(_x, _n) (uint8_t)(((_x) >> ((_n)*8)) & 0x00FF)
```

Comment:
migrate to `macros.h`?

Reply:
Yeah, that's in progress in what I'm working on right now, gonna push a new commit with it

Reply:
Moved this into macros.h. Also changed the macros to be
`NTOHS` = Network-to-host short (16-bit)
`NTOHL` = Network-to-host long (32-bit)
 `HTONS` = Host-to-network short (16-bit)
 `HTONL` = Host-to-network long (32-bit)